src/components/Table/Table.tsx

```diff
@@ -44,8 +44,6 @@ export interface Props {
 }
 
 /**
- * BETA: This component is still a work in progress and is subject to change.
- *
```

Comment:
Can we keep beta for Table and TableObject as well since they're being formally announced with 1.1?